src/ln/channelmonitor.rs

```diff
@@ -475,6 +486,36 @@ impl ChannelMonitor {
 		}
 	}
 
+	fn get_witnesses_weight(inputs: &Vec<InputDescriptors>) -> u64 {
+		let mut tx_weight = 0;
```

Comment:
Doesn't this need to start with 2 for the flags bytes?

Reply:
Aaaah in fact I was adding them in test to get the right count but it didn't trigger me that it wasn't normal and that I should get them from get_witnesses_weight....